src/components/forms/common/FileUploader.tsx

```diff
@@ -3,33 +3,26 @@ import { Controller } from 'react-hook-form';
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore TODO: fix import errors
 import { MuiFileInput } from 'mui-file-input';
-import { useState } from 'react';
 
 type Props = {
 	name: string;
 	control: Control<any>;
 	label?: string;
 };
 
-//TODO: make image loading to work (value not passed via context now)
-export const FileUploader = ({ name, control, label }: Props) => {
-	const [file, setFile] = useState<null | File>(null);
-	return (
-		<Controller
-			name={name}
-			control={control}
-			render={({ field, fieldState }) => (
-				<MuiFileInput
-					{...field}
-					label={label}
-					value={file}
-					onChange={(newFile: File | null) => {
-						setFile(newFile);
-					}}
-					helperText={fieldState.invalid ? 'File is invalid' : ''}
-					error={fieldState.invalid}
-				/>
-			)}
-		/>
-	);
-};
+export const FileUploader = ({ name, control, label }: Props) => (
+	<Controller
```

Comment:
Dokázal bys tam ještě přidat konverzi dat při nahrání, aby se to nenačetlo jako File, ale rovnou jako pole bytů?
Mělo by to jít pomocí nějaké metody: https://developer.mozilla.org/en-US/docs/Web/API/File
Firebase pak normálně podporuje typ bytes: https://firebase.google.com/docs/firestore/manage-data/data-types